web-access-app/src/main/java/edu/unc/lib/boxc/web/access/controllers/FullRecordController.java

```diff
@@ -27,6 +27,7 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import edu.unc.lib.boxc.web.common.services.XmlDocumentFilteringService;
```

Comment:
Import looks like it's in the wrong place

Reply:
I think it is a disagreement between eclipse and intellij about the sort order of the imports, but I've got ahead and had intellij organize the imports